meilisearch-http/tests/dumps/data.rs

```diff
@@ -0,0 +1,51 @@
+use std::path::PathBuf;
+
+use manifest_dir_macros::exist_relative_path;
+
+pub fn get_v1_movies_raw() -> PathBuf {
```

Comment:
Could you please define an enum here and define only one function?

Reply:
done